packages/blade/src/components/Spinner/BaseSpinner/BaseSpinner.native.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Animated, {
+  cancelAnimation,
+  useAnimatedStyle,
+  useSharedValue,
+  withRepeat,
+  withTiming,
+} from 'react-native-reanimated';
+import getIn from '../../../utils/getIn';
+import { useTheme } from '../../BladeProvider';
+import BaseLoader from './BaseLoader';
+import type { BaseSpinnerProps } from './BaseSpinner.d';
+
+type WithStyle = {
+  style: Record<string, unknown>;
+};
+type AnimatedBaseSpinnerProps = {
+  children: React.ReactNode;
+} & WithStyle;
+
+const AnimatedBaseSpinner = (props: AnimatedBaseSpinnerProps): React.ReactElement => {
+  const { theme } = useTheme();
+  const duration = getIn(theme.motion, 'duration.2xgentle');
+  const easing = getIn(theme.motion, 'easing.standard.effective');
```

Comment:
Lets store these as component-level tokens. Want to ensure that we can make changes in 1 place for easing & duration values for spinner and it should reflect on native & web both